### ISSUE TITLE
Enhance linting

### ISSUE THREAD

**Issue by alessandroargentieri** [gyozatech/kannon#52]
**Bug description**

Repair linting issues as described into the report:

https://goreportcard.com/report/github.com/gyozatech/kannon

This would generate an higher score for the code quality tools.

**Comment by ludusrusso** [comment]
Yep, I think what we need is to ad and action here that checks for output of golint!
This would be helpfull
